app/components/support_interface/reasons_for_rejection_table_component.rb

```diff
@@ -0,0 +1,30 @@
+module SupportInterface
+  class ReasonsForRejectionTableComponent < ViewComponent::Base
+    include ViewHelper
+    attr_accessor :reasons
+
+    TOP_LEVEL_REASONS_I18N_KEYS = {
+      candidate_behaviour_y_n: :candidate_behaviour_what_did_the_candidate_do,
+      quality_of_application_y_n: :quality_of_application,
+      qualifications_y_n: :qualifications_which_qualifications,
```

Comment:
Should this be `qualifications`?

Reply:
I'm a bit confused about these names to be honest but the sub-reason labels were put under the `qualifications_which_qualifications` key so this works even if it seems a bit inconsistent.